datanode/api/trading_data_v2.go

```diff
@@ -1418,11 +1418,11 @@ func (t *tradingDataServiceV2) ListWithdrawals(ctx context.Context, req *v2.List
 // -- Assets --.
 func (t *tradingDataServiceV2) GetAsset(ctx context.Context, req *v2.GetAssetRequest) (*v2.GetAssetResponse, error) {
 	defer metrics.StartAPIRequestAndTimeGRPC("GetAssetV2")()
-	if req == nil || req.AssetId == "" {
+	if req == nil || req.Id == "" {
```

Comment:
Did you mean to leave this one in?

Reply:
No,
One of the issue points was to make the objects able to be queried by ID, not by "objectId"; and specifically asset needed to become -> Asset { Id: ID } instead of Asset { AssetId: String }, etc